t/05-distribution.t

```diff
@@ -59,6 +59,58 @@ subtest 'script_run' => sub {
     $d->script_run('long_command' x 512);
     $cmdcall = $wait_serial_calls[1];
     is $cmdcall->{buffer_size}, 6272, 'appropriate buffer size used for long command';
+    undef($d);
+};
+
+subtest 'set expected serial and autoinst failures' => sub {
+    require distribution;
```

Comment:
It would make more sense to `require` or `use` this once at the top instead of repeating this line.